source/codegen/proto_helpers.py

```diff
@@ -1,12 +1,11 @@
 import common_helpers
 
-def should_allow_alias(enums):
-  if enums.get("generate-mappings", False):
+def should_allow_alias(enum):
```

Comment:
Note: the above code can be removed because this code has been updated to handle the duplicate-non-zero values. we previously had 2 versions of this code that each handled a different special case.